roles/cifmw_setup/tasks/deploy_edpm.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Run pre_deploy hooks
+  when:
+    - cifmw_architecture_scenario is not defined
```

Comment:
I see all conditions here are similar: `when: cifmw_architecture_scenario is not defined`.
Maybe we should consider to add it as a one block?